### ISSUE TITLE
[Redesign beta] Can't see long song titles because two-rows max

### ISSUE THREAD

**Issue by mingussun** [jmshrv/finamp#995]
Before it used as many rows as needed to display the full title. Would love if that was brought back if possible. Perhaps by enabling it in settings? Not seeing the full title at a glance is often a problem with classical music and makes it a pain if you want a specific track. 

![IMG_0111](https://github.com/user-attachments/assets/fd345af9-58dd-4e98-a8da-bf17fa1d6abe)

**Comment by Chaphasilor** [comment]
Since 0.9.13 or 0.9.14 you should be able to enable scrolling text / marquee in the track menu. So as a workaround you could long-press a track to open the menu, and see the full title there.  
I'm honestly not sure how this issue could be fixed; having tons of track scrolling their titles at the same time seems terrible to me.

Maybe better support for classical music, like suggested in #613, would help here?

**Comment by mingussun** [comment]
I’m not a fan of scrolling text personally, but it’s definitely better than nothing. I agree marquee text for all tracks at the same time would be terrible. 

Ideally for me it would be like it was before (see old screenshot). Sure I guess it might not look great, but actually being able to read the song titles (without having to wait for text to scroll) takes precedence for me. And it’s not an issue for albums with ”normal” titles. 

Would it be possible to have it as a setting to enable more rows for song titles if needed, or something like that? I have no idea, so I always assume it’s not easy. 

I appreciate all the devs and I love the app and redesign in general.

![Image](https://github.com/user-attachments/assets/7ab68417-5c2b-495b-9178-72164edb2cae)

**Comment by Chaphasilor** [comment]
Yeah I think that might be possible, although it could cause issues with scrolling performance since not all tracks will have the same height any more.
